tests/unit/test_network_connections.py

```diff
@@ -2998,6 +2998,106 @@ def test_valid_persistent_state(self):
         input_connection.update({"state": "absent"})
         self.assertValidationError(validator, input_connection)
 
+    def test_set_deprecated_master(self):
+        """
+        When passing the deprecated "master" it is updated to "controller".
+        """
+        input_connections = [
+            {
+                "name": "prod2",
+                "state": "up",
+                "type": "bridge",
+                "interface_name": "bridge2",
+                "ip": {"dhcp4": False, "auto6": False},
+            },
+            {
+                "name": "prod2-slave1",
```

Comment:
the name can use inclusive language, too.

Reply:
Oops. It should, I just copied it from another example. The example will use inclusive language on #303, I will fix the new one.